README.md

```diff
@@ -307,6 +307,34 @@ add_zarr_level(
     compressor=numcodecs.Blosc(cname="zstd", clevel=3, shuffle=numcodecs.Blosc.BITSHUFFLE)
 )
 ```
+### Using Config Presets
+
+Config presets make it easy to get started with `OMEZarrWriter` without needing to know all of its options. They inspect your input data and return a configuration dictionary that you can pass directly into the writer.
+
+#### Visualization preset
+
+The visualization preset (`get_default_config_for_viz`) creates a multiscale pyramid (full resolution plus downsampled levels along Y/X) suitable for interactive browsing.
+
+```python
+import numpy as np
+from bioio_ome_zarr.writers import (
+    OMEZarrWriter,
+    get_default_config_for_viz,
+)
+
+data = np.zeros((1, 1, 4, 64, 64), dtype="uint16")
```

Comment:
is writing from raw data the most likely use case?  I suppose the input file data gets manipulated somehow, or you'd just use a direct conversion tool.  Just wondering if a input czi-> output zarr would be helpful here.